modules/foreman_compute_resource.py

```diff
@@ -39,7 +46,11 @@
     default: None
     choices: ["vmware", "libvirt", "ovirt"]
   provider_auth:
-    description: provider authentication
+    description: Deprecated, use provider_params instead. Provider authentication
```

Comment:
Would you mind using `I()` for the option name?  `I(provider_params)`

Reply:
@bagasse I think you missed this one? :-)

Reply:
@sean797 ooops.... sorry... Done :)